examples/run_var_tau.py

```diff
@@ -76,8 +77,8 @@
     ret_data=None,
 )
 mytau_axis = var_tau_data.getaxis("indirect")
-mytau_axis[0]["tau_adjust"] = tau_adjust
-mytaus_axis[0]["tau"] = tau
+mytau_axis[0]["tau_adjust"] = tau_adjust_range[0]
+mytau_axis[0]["tau"] = tau[0]
```

Comment:
What is going on here? This is a non-rhetorical question to be replied to in the pr